fxslack/README.md

```diff
@@ -0,0 +1,69 @@
+# Fx Slack Module
+
+[![ci](https://github.com/ankorstore/yokai-contrib/actions/workflows/fxslack-ci.yml/badge.svg)](https://github.com/ankorstore/yokai-contrib/actions/workflows/fxslack-ci.yml)
+[![go report](https://goreportcard.com/badge/github.com/ankorstore/yokai-contrib/fxslack)](https://goreportcard.com/report/github.com/ankorstore/yokai-contrib/fxslack)
+[![codecov](https://codecov.io/gh/ankorstore/yokai-contrib/graph/badge.svg?token=ghUBlFsjhR&flag=fxslack)](https://app.codecov.io/gh/ankorstore/yokai-contrib/tree/main/fxslack)
+[![Deps](https://img.shields.io/badge/osi-deps-blue)](https://deps.dev/go/github.com%2Fankorstore%2Fyokai-contrib%2Ffxslack)
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/ankorstore/yokai-contrib/fxslack)](https://pkg.go.dev/github.com/ankorstore/yokai-contrib/fxslack)
+
+> [Fx](https://uber-go.github.io/fx/) module for [Slack](https://api.slack.com).
+
+<!-- TOC -->
+* [Overview](#overview)
+* [Installation](#installation)
+* [Configuration](#configuration)
+* [Health Check](#health-check)
+* [Testing](#testing)
+<!-- TOC -->
+
+## Overview
+
+This module provides to your Fx application a [slack.Client](https://pkg.go.dev/github.com/slack-go/slack#Client),
+that you can `inject` anywhere to interact with the Slack API.
+
+## Installation
```

Comment:
we need to mention that this module requires the `fxhttpclient` module to be installed, as otherwise the required `http.RoundTripper` will be missing in DI, preventing Fx to boot.